services/opencollective/opencollective-by-tier.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+
+// https://developer.opencollective.com/#/api/collectives?id=get-members-per-tier
+const membersArraySchema = Joi.array().items(
+  Joi.object().keys({
+    MemberId: Joi.number().required(),
+    tier: Joi.string().required(),
+    role: Joi.string().required(),
+  })
+)
+
+module.exports = class OpencollectiveByTier extends BaseJsonService {
+  static get category() {
+    return 'funding'
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'open collective members by tier',
+        namedParams: { collective: 'shields', tierSlug: 'monthly-backer' },
+        staticExample: this.render('shields', 'monthly-backer', 8),
+        keywords: ['opencollective'],
+      },
+    ]
+  }
+
+  static get route() {
+    return {
+      base: 'opencollective/tier',
+      pattern: ':collective/:tierSlug',
```

Comment:
Good question. I should've thought about that. They are not standardized - you can add customized ones. I think the slug is the lowercased title of the tier with spaces replaced with hyphens. Maybe one of the collective administrators could have a look at the tier-settings in the collective and check if they can find anything on that.
I just noticed that I should also add the possibility to identify the tier by its id. Would the best way to do this be by adding another badge?
I'll also add the docs.

Reply:
I think so, yes. I didn't find any official information from open collective yet though.

Reply:
I briefly looked at this badge a while back and saved some doc links. I found the content out here: https://developer.opencollective.com/#/api/collectives?id=get-members pretty helpful

Reply:
Tier slugs info (I think): https://developer.opencollective.com/#/api/collectives?id=get-members-per-tier

Reply:
The tier IDs do show up in the UI. Maybe the tier badge should use that other endpoint so it can use that ID instead.

Reply:
That just fortifies our assumption but doesn't really confirm it.
I'll definitely add the possibility to identify the tier by its id.

Reply:
> The tier IDs do show up in the UI. Maybe the tier badge should use that other endpoint so it can use that ID instead.

 Should we only use the IDs and drop support for the slug?

Reply:
Yea, I think that makes sense.

Reply:
The only other tier related content I could find was https://github.com/opencollective/opencollective/wiki/Adding-tiers-to-your-Collective but it doesn't shed any additional light on what's possible for valid tier name values, etc. 

Tier Id seems like the better bet

Reply:
That's unfortunate… I think we could show **new tier** as a fallback in that case.

Reply:
Showing _new tier_ sounds good. I'll also add this information to the documentation for finding the tier ID.